src/main/java/com/salesforce/dev/pages/Product/ProductsHome.java

```diff
@@ -4,30 +4,32 @@
 import org.openqa.selenium.Alert;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
+import com.salesforce.dev.pages.Base.*;
 /**
  * Created by marcelo on 6/11/2015.
  */
-public class ProductsHome {
+public class ProductsHome extends TabPage  {
     WebDriver driver;
     WebDriverWait wait;
 
```

Comment:
We can delete the driver and wait because in the abstract class they are already declared